### DISCUSSION TITLE
0.22.0

### DISCUSSION THREAD

**Discussion by tillsteinbach** [tillsteinbach/VWsFriend#334] [General]
### Added
- Allow to hide and sort vehicles in the grafana select box through vehicle settings
- Tire warning light category

### Fixed
- Bug with honk and flash endpoint

### Changed
- Updated API to 0.48.1
- Updated Grafana to 9.1.6

<hr /><em>This discussion was created from the release <a href='https://github.com/tillsteinbach/VWsFriend/releases/tag/v0.22.0'>0.22.0</a>.</em>

**Comment by marcofuchs89** [comment]
Hi (@tillsteinbach) Till,

after upgrading to the version 0.22.0 the container keep's restarting over and over again.
The log shows a stacktrace to an error in the Python Threading module, that no new thread can be started:

```python
Traceback (most recent call last):
vwsfriend_1   |   File "/opt/venv/bin/vwsfriend", line 8, in <module>
vwsfriend_1   |     sys.exit(main())
vwsfriend_1   |   File "/opt/venv/lib/python3.10/site-packages/vwsfriend/vwsfriend_base.py", line 458, in main
vwsfriend_1   |     ui.run(host=args.host, port=args.port)
vwsfriend_1   |   File "/opt/venv/lib/python3.10/site-packages/vwsfriend/ui/vwsfriend_ui.py", line 209, in run
vwsfriend_1   |     webthread.start()
vwsfriend_1   |   File "/usr/lib/python3.10/threading.py", line 928, in start
vwsfriend_1   |     _start_new_thread(self._bootstrap, ())
vwsfriend_1   | RuntimeError: can't start new thread
```
Do you have any advice on how to fix that?

Best regards,
Marco

**Comment by tillsteinbach** [comment]
Hi Marco, can you doublecheck that you have at least Docker 20.10.10 or later?

Cheers!
Till

**Comment by marcofuchs89** [comment]
Hi Till,

good catch. I‘m running 20.10.8 on my server. I‘ll get that fixed and test again.

Thanks for your quick reply! 👍

Cheers
Marco
Am 29. Sept. 2022, 21:00 +0200 schrieb Till Steinbach ***@***.***>:

**Reply by marcofuchs89** [reply]
Hi Till,

I've upgraded my docker engine, and it's now working again 👍🏼

One further question: There's now a timeframe, where there was no data imported - is there any config switch that can be toggled to import historical data?

Best regards and a nice weekend,
Marco

**Reply by tillsteinbach** [reply]
No, unfortunately there is no history. That is why VWsFriend has to run 24/7 to get all data.
